src/test/components/__snapshots__/FooterLinks.test.js.snap

```diff
@@ -0,0 +1,119 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`correctly renders the FooterLinks component 1`] = `
+<body>
+  <div>
+    <div
+      class="appFooterLinks"
+    >
+      <button
+        aria-label="Hide footer links"
+        class="appFooterLinksClose"
+        title="Hide footer links"
+        type="button"
+      >
+        ✕
+      </button>
+      <a
+        class="appFooterLinksLink"
+        href="https://www.mozilla.org/about/legal/terms/mozilla"
+        rel="noopener noreferrer"
+        target="_blank"
+      >
+        Legal
+      </a>
+      <a
+        class="appFooterLinksLink"
+        href="https://www.mozilla.org/privacy/websites"
+        rel="noopener noreferrer"
+        target="_blank"
+      >
+        Privacy
+      </a>
+      <a
+        class="appFooterLinksLink"
+        href="https://www.mozilla.org/privacy/websites/#cookies"
+        rel="noopener noreferrer"
+        target="_blank"
+      >
+        Cookies
+      </a>
+      <select
+        class="appFooterLinksLanguageSwitcher"
+        title="Change language"
+      >
+        <option
+          value="de"
+        >
+          Deutsch
+        </option>
+        <option
+          value="el"
+        >
+          Ελληνικά
+        </option>
+        <option
+          value="en-GB"
+        >
+          English (GB)
+        </option>
+        <option
+          value="en-US"
+        >
+          English (US)
+        </option>
+        <option
+          value="es-CL"
```

Comment:
I'm just curious, why is this chilean spanish?

Reply:
Because this is the locale we have :-) https://pontoon.mozilla.org/projects/firefox-profiler/

Reply:
Also please look at this line in the diff: https://github.com/firefox-devtools/profiler/pull/3905/files#diff-07d8e2d52e3f9d17b1eed19cb0c080fafd97555afe0dbfb3d591acaa7794a351R26

I decided (arbitrarily) that because this is the only spanish flavor we have we can label it as simply `Español` until we have more flavors. Flod seemed to agree but I'd be happy to revisit this decision.

Reply:
Yeah, seems reasonable to me. `es` is a fallback from `es-CL`. `es-CL` seems fairly specific to support, where I would expect more of a `es-ES` and `es-419` breakdown. The latter is the "Latin American Spanish", where 419 is a [region code](https://en.wikipedia.org/wiki/UN_M49). However, any Spanish translation here is probably pretty readable, as the main UI's vocab is quite niche and surprising anyway.

Reply:
We're depending on our contributors for the available translations :-)